modules/exploits/linux/local/exim4_deliver_message_priv_esc.rb

```diff
@@ -221,12 +221,6 @@ def check
   end
 
   def exploit
-    if is_root?
-      unless datastore['ForceExploit']
-        fail_with(Failure::BadConfig, 'Session already has root privileges. Set ForceExploit to override.')
-      end
-    end
```

Comment:
The `is_root?` check should remain.

Reply:
Ok, I will be sure to get that added back in.